specs/mcpfile-schema-0.1.0.json

```diff
@@ -0,0 +1,477 @@
+          "description": "Configuration for streamable HTTP transport protocol."
+        },
+        "stdioConfig": {
+          "$ref": "#/$defs/StdioConfig",
+          "description": "Configuration for stdio transport protocol."
+        }
+      },
+      "additionalProperties": false,
+      "type": "object",
+      "required": [
+        "transportProtocol"
+      ],
+      "description": "ServerRuntime defines transport protocol and associated configuration."
+    },
+    "StdioConfig": {
+      "properties": {},
+      "additionalProperties": false,
+      "type": "object",
+      "description": "StdioConfig defines configuration for stdio transport protocol."
+    },
+    "StreamableHTTPConfig": {
+      "properties": {
+        "port": {
+          "type": "integer",
+          "description": "Port number to listen on."
+        },
+        "basePath": {
+          "type": "string",
+          "description": "Base path for the MCP server (default: /mcp)."
+        },
+        "stateless": {
+          "type": "boolean",
+          "description": "Indicates whether the server is stateless."
+        },
+        "auth": {
+          "$ref": "#/$defs/AuthConfig",
+          "description": "OAuth 2.0 configuration for protected resources."
+        },
+        "tls": {
+          "$ref": "#/$defs/TLSConfig",
+          "description": "TLS configuration for HTTPS."
+        }
+      },
+      "additionalProperties": false,
+      "type": "object",
+      "required": [
+        "port",
+        "stateless"
+      ],
+      "description": "StreamableHTTPConfig defines configuration for the HTTP-based runtime."
+    },
+    "TLSConfig": {
+      "properties": {
+        "certFile": {
+          "type": "string",
+          "description": "Absolute path to the server's public certificate."
+        },
+        "keyFile": {
+          "type": "string",
+          "description": "Absolute path to the server's private key."
+        }
+      },
+      "additionalProperties": false,
+      "type": "object",
+      "description": "TLSConfig defines paths to TLS certificate and private key files."
+    },
+    "Tool": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Unique identifier for the tool."
+        },
+        "title": {
+          "type": "string",
+          "description": "Human-readable title for display purposes."
+        },
+        "description": {
+          "type": "string",
+          "description": "Detailed description of the tool's purpose."
+        },
+        "inputSchema": {
+          "$ref": "#/$defs/Schema",
+          "description": "JSON Schema describing input parameters."
+        },
+        "outputSchema": {
+          "$ref": "#/$defs/Schema",
+          "description": "Optional JSON Schema describing output."
+        },
+        "invocation": {
+          "description": "Object describing how to execute the tool."
+        },
```

Comment:
Same comment as for the prompt invocation

Reply:
Done now